lwa4d_moveit_config/package.xml

```diff
@@ -0,0 +1,29 @@
+<package>
+
+  <name>lwa4d_moveit_config</name>
+  <version>0.2.0</version>
+  <description>
+     An automatically generated package with all the configuration and launch files for using the lwa4d with the MoveIt Motion Planning Framework
+  </description>
+  <author email="assistant@moveit.ros.org">MoveIt Setup Assistant</author>
+  <maintainer email="assistant@moveit.ros.org">MoveIt Setup Assistant</maintainer>
```

Comment:
I will start now trying to use the moveit config in demo model/with simulation/with hardware... I let you know as soon as I have some feedback of my results. How can I add the author and maintainer info?

About the github-user I dont know why I used my private github account. Do you have any idea why? I did the pull request using my ipa-bfb account.